event_track_location_overlap/README.rst

```diff
@@ -56,6 +56,7 @@ Contributors
 * `Tecnativa <https://www.tecnativa.com>`_:
 
   * Jairo Llopis <jairo.llopis@tecnativa.com>
+  * Cristina Martin R.
```

Comment:
No need to add this if you split the README. It will be added on next autogeneration by bot.